src/index.js

```diff
@@ -1 +1,41 @@
-// Your server code here...
+import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import contactRoutes from './routes/ContactRoutes';
+// Can also use const express = require('express');
+
+
+// Create a new instance of express
+const app = express();
+
+
+// Connect to the mongo database
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/contact-list');
+
+
+// Middleware that parses incoming request bodies, and makes the payload
+// available to use through request.body
+app.use(bodyParser.json());
+
+
+// Add ContactRoutes to application
+app.use(contactRoutes);
+
+app.use(function (err, request, response) {
```

Comment:
This middleware isn't going to work correctly because it needs 4 arguments, and we've only supplied 3.

`(err, req, res, next)`

We need to turn off the rule for no unused vars for this